recipes/systemc/meta.yaml

```diff
@@ -0,0 +1,146 @@
+          - lib/pkgconfig/tlm.pc                                 # [not win]
+          - lib/cmake/SystemCLanguage/                           # [not win]
+          - lib/cmake/SystemCTLM/                                # [not win]
+          - include/sysc                                         # [not win]
+          - include/systemc*                                     # [not win]
+          - include/tlm*                                         # [not win]
+          - include/tlm_core                                     # [not win]
+          - include/tlm_utils                                    # [not win]
+          - etc/conda/*activate.d/*activate-systemc.sh           # [not win]
+      test:                                                      # [not win]
+        requires:                                                # [not win]
+          - {{ compiler('cxx') }}                                # [not win]
+          - {{ pin_subpackage('systemc-doc', exact=True) }}      # [not win]
+          - make                                                 # [not win]
+          - sed                                                  # [not win]
+          - pkg-config                                           # [not win]
+        commands:                                                # [not win]
+          - test "$SYSTEMC_HOME" == "$PREFIX"                    # [not win]
+          - ls -l "$PREFIX"/lib                                  # [not win]
+          - test -f "$PREFIX"/lib/libsystemc"$SHLIB_EXT"         # [not win]
+          - test -f "$PREFIX"/lib/libsystemc"$SHLIB_EXT".{{ major_minor }}  # [linux]
+          - test -f "$PREFIX"/lib/libsystemc.{{ major_minor }}"$SHLIB_EXT"  # [osx]
+          - pkg-config --print-provides systemc                             # [not win]
+          - pkg-config --print-errors --exact-version=$PKG_VERSION systemc  # [not win]
+          - pkg-config --print-provides tlm                                 # [not win]
+          # tlm version is decoupled from systemc, so just make sure pkg-config is there
+          - cp -r "$PREFIX"/share/doc/systemc/examples .         # [not win]
+          # patch Makefile.rules because we want to use the installed libsystemc  
+          # AND we want to use the dynamic lib which the linker should be able to find
+          - "sed -i 's|$(EXE): $(OBJS) $(SYSTEMC_LIB_DIR)/libsystemc.a|$(EXE): $(OBJS)|' examples/build-unix/Makefile.rules"  # [not win]
+          - make -C examples/build-unix -j $CPU_COUNT run        # [linux]
+          - make -C examples/build-unix -j $CPU_COUNT check      # [linux]
+          # osx ld64 doesn't support '-rpath=' must be '-rpath,'
+          - make -C examples/build-unix -j $CPU_COUNT run LDFLAG_RPATH=-Wl,-rpath,    # [osx]
+          - make -C examples/build-unix -j $CPU_COUNT check LDFLAG_RPATH=-Wl,-rpath,  # [osx]
+    - name: systemc                                              # [not win]
```

Comment:
Each output that could possibly be a direct build dependency for downstream packages needs to have a run export for a compatible version of libsystemc. This includes, systemc, systemc-dev, and libsystemc.